### PR TITLE
feat: update for node context update

### PR DESCRIPTION
**Pull request by frdomovic**
# feat: update for node context update

### CODE REVIEW

File: app/src/api/dataSource/ClientApiDataSource.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class ClientApiDataSource implements ClientApi {
 
     const response = await getJsonRpcClient().query<FeedRequest, Post[]>(
       {
-        applicationId: applicationId,
+        contextId: contextId,
```
Comment (miraclx) [inline app/src/api/dataSource/ClientApiDataSource.ts:39]:
```suggestion
        contextId,
```

File: app/src/api/dataSource/ClientApiDataSource.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export class ClientApiDataSource implements ClientApi {
 
     const response = await getJsonRpcClient().query<PostRequest, Post>(
       {
-        applicationId: applicationId,
+        contextId: contextId,
```
Comment (miraclx) [inline app/src/api/dataSource/ClientApiDataSource.ts:63]:
```suggestion
        contextId,
```

File: app/src/api/dataSource/ClientApiDataSource.ts [hunk 3]
```diff
@@ -83,7 +83,7 @@ export class ClientApiDataSource implements ClientApi {
 
     const response = await getJsonRpcClient().mutate<CreatePostRequest, Post>(
       {
-        applicationId: applicationId,
+        contextId: contextId,
```
Comment (miraclx) [inline app/src/api/dataSource/ClientApiDataSource.ts:86]:
```suggestion
        contextId,
```

File: app/src/api/dataSource/ClientApiDataSource.ts [hunk 4]
```diff
@@ -109,7 +109,7 @@ export class ClientApiDataSource implements ClientApi {
       Comment
     >(
       {
-        applicationId: applicationId,
+        contextId: contextId,
```
Comment (miraclx) [inline app/src/api/dataSource/ClientApiDataSource.ts:112]:
```suggestion
        contextId,
```

File: app/src/crypto/crypto.ts [hunk 5]
```diff
@@ -37,6 +39,7 @@ export async function createAuthHeader(
     signing_key: signing_key,
     signature: signatureBase58,
     challenge: contentBase58,
+    contextId: contextId
```
Comment (miraclx) [inline app/src/crypto/crypto.ts:42]:
```suggestion
    contextId,
```

File: app/.env.production [hunk 6]
```diff
@@ -1,4 +1,4 @@
 NEXT_PUBLIC_NEAR_ENV=testnet
-NEXT_PUBLIC_APPLICATION_ID="5d77b7852d5c2fa9483ece70ada7e3cc7d2a9df5aa056d3916103dbe783df6e2"
+NEXT_PUBLIC_CONTEXT_ID="5d77b7852d5c2fa9483ece70ada7e3cc7d2a9df5aa056d3916103dbe783df6e2"
```
Comment (miraclx) [inline app/.env.production:2]:
```suggestion
NEXT_PUBLIC_CONTEXT_ID="wAazFnyNS7eYQSjCAxG25nyxaUywUmMt7i92KmtUTyV"
```

**Review by miraclx — APPROVED** [review]
lgtm

**Comment by frdomovic** [comment]
All comments updated

**Pull request merged**